corehq/apps/locations/views.py

```diff
@@ -270,11 +275,13 @@ def data_sources(self):
 @method_decorator(locations_access_required, name='dispatch')
 @location_safe
 class LocationsSearchView(EmwfOptionsView):
+    include_locations_with_no_users_allowed = True
```

Comment:
Wouldn't this be true regardless of the endpoint? So it'll be `True` even for `location_search`?

Reply:
ah, i just forgot `as_view` will instantiate the class attribute. I was thinking it had to be done somewhere here.